### DISCUSSION TITLE
Can I use 16px version only and disable switching to 24px version?

### DISCUSSION THREAD

**Discussion by orcunilbeyli** [primer/octicons#741] [Q&A]
I want to use 16px and control the resizing by myself. In other words, I don't want icons to get thinner if I increase the size. When I look at the repo/code it looks like hardcoded or am I missing anything? Thanks for the great work btw!

**Comment by KaAwatea** [comment]
I hope you got an answer.  I'm thinking, the settings to reduce text size or expand your page from the corner of your message box(es).  All the best!!
